storm-core/test/clj/org/apache/storm/scheduler_test.clj

```diff
@@ -262,20 +262,15 @@
 
 (deftest test-sort-slots
   ;; test supervisor2 has more free slots
-  (is (= '(["supervisor2" 6700] ["supervisor1" 6700]
-           ["supervisor2" 6701] ["supervisor1" 6701]
-           ["supervisor2" 6702])
-         (sort-slots [["supervisor1" 6700] ["supervisor1" 6701]
-                      ["supervisor2" 6700] ["supervisor2" 6701] ["supervisor2" 6702]
-                      ])))
+  (is (= "[supervisor2:6700, supervisor1:6700, supervisor2:6701, supervisor1:6701, supervisor2:6702]"
+         (.toString (EvenScheduler/sortSlots [(WorkerSlot. "supervisor1" 6700) (WorkerSlot. "supervisor1" 6701)
```

Comment:
Can we remove the .toString and compare against a List of WorkerSlots?  We might need a clojurifyStructure in there  too?

i.e

```
(is (= [(WorkerSlot. "supervisor2" 6700) (WorkerSlot. "supervisor1" 6700) 
          (WorkerSlot. "supervisor2" 6701) (WorkerSlot. "supervisor1" 6701)
          (WorkerSlot. "supervisor2" 6702)]
    (clojurify-structure (EvenScheduler/sortSlots [(WorkerSlot. "supervisor1" 6700) (WorkerSlot. "supervisor1" 6701)...
```